crates/store/re_types/definitions/rerun/blueprint/archetypes/dataframe_settings.fbs

```diff
@@ -7,17 +7,20 @@ include "rerun/attributes.fbs";
 namespace rerun.blueprint.archetypes;
 
 
-/// Configuration for the sorting of the rows of a time range table.
-table RangeTableOrder (
+/// Configuration for the dataframe view
+table DataframeSettings (
```

Comment:
I'd be very wary of calling this `DataframeSettings` -- "Dataframe" is already way too overloaded.

```suggestion
table DataframeViewSettings (

Reply:
Actually, after discussing with Andreas, I'm going to split that archetype in two, more focused archetypes. We need that anyways to prep for the blueprint API.

Reply:
Also for record, I'm not a fan of the "Dataframe" name for that view. Happy to change it. My current fav is "DataBrowserView" (assuming we pour every single feature in that specific view, with mode, etc.—as opposed to split them across multiple view kinds).